### ISSUE TITLE
Parse JSON output into scalar vars, support a prefix

### ISSUE THREAD

**Issue by sevaa** [jikuma/AzureDevOpsGithubRestApi#31]
Who needs the whole JSON from the GET. Individual values is what one needs. Also, since the same (or similar) API could be called more than once in a pipeline, the task needs a variable prefix parameter to disambiguate.

 So, if the result JSON goes
```
{
   "a": "Foo"
   "b": "Bar"
   "c": {"d": "Baz"}
   "e": [10,20]
}
```
And the prefix parameter is "my_", the task, I think, should publish the following variables:

my_a = Foo
my_b = Bar
my_c = {"d": "Baz"}
my_c.d = Baz
my_e.length = 2
my_e.0 = 10
my_e.1 = 20

May I send a PR to that effect?

**Comment by jikuma** [comment]
If you raise a PR, I will love to review it and update the extension.

**Comment by jikuma** [comment]
Thanks, @sevaa for contributing. I have published the updated task with version 2.1.
